app/views/govuk_publishing_components/components/_summary_list.html.erb

```diff
@@ -88,12 +86,10 @@
 
             <% if item.fetch(:delete, {}).any? %>
               <% delete_link = capture do %>
-                <%
-                  delete_link_text = item[:delete][:link_text] || t("components.summary_list.delete")
-                %>
+                <% delete_link_text = item[:delete][:link_text] || t("components.summary_list.delete") %>
                 <%= link_to item[:delete].fetch(:href),
-                            class: "govuk-link gem-link--destructive",
-                            data: item[:delete].fetch(:data_attributes, {}) do %>
+                      class: "govuk-link gem-link--destructive",
+                      data: item[:delete].fetch(:data_attributes, {}) do %>
                   <%= delete_link_text %><%= tag.span " #{item[:field]}", class: "govuk-visually-hidden" unless item[:delete][:link_text_no_enhance] -%>
```

Comment:
Only a suggestion that perhaps this could be written a little differently so that the indentation, and the `<% end %>` of the block are a little easier to identify. Something like:

```
  <%= link_to delete.fetch(:href), class: "govuk-link gem-link--destructive", data: delete.fetch(:data_attributes, {}) do %>
    <%= delete_section_link_text %><%= tag.span " #{title}", class: "govuk-visually-hidden" unless delete[:link_text_no_enhance] -%>
  <% end %>
```

Same question / comment for line 78 too with regards to the edit link.

Reply:
Let me capture this one as it's tad out of scope.